docs/root/start/quick-start/_include/envoy-demo-tls-validation.yaml

```diff
@@ -46,6 +46,7 @@ static_resources:
       name: envoy.transport_sockets.tls
       typed_config:
         "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
+        sni: proxy-postgres-backend.example.com
```

Comment:
I added this so that it's a more complete example for how someone should actually configure this

Reply:
im not sure i agree with this change - this is an example _not_ using sni - also this breaks the highlighting

Reply:
Want me to revert this one? No SNI with validation is fine. Anything with no validation is probably not-fine.